server/server.go

```diff
@@ -462,6 +465,8 @@ func getNetworkBlockchainSpec(
 }
 
 func (s *server) CreateBlockchains(ctx context.Context, req *rpcpb.CreateBlockchainsRequest) (*rpcpb.CreateBlockchainsResponse, error) {
+	s.mu.Lock()
```

Comment:
We're having the same pattern here, a lock, then unlock before every return, *except* one guy. Suspicious IMHO...